templates/cicd/pipeline_cfn.yml

```diff
@@ -233,29 +232,44 @@ Resources:
       RoleArn: !GetAtt PipelineRole.Arn
       Name: !Ref AWS::StackName
       Stages:
+        {{- if eq .Source.ProviderName "GitHub"}}
         - Name: Source
           Actions:
             - Name: SourceCodeFor-{{$.AppName}}
               ActionTypeId:
                 Category: Source
-                # TODO: #273 For CodeCommit, this needs to be AWS. Let's do that later
                 Owner: ThirdParty
                 Version: 1
-                Provider: {{.Source.ProviderName}}
+                Provider: GitHub
               Configuration:
-                # TODO: #273 For now this set of configurations is GitHub-specific,
-                # change it to be more generic
                 Owner: {{$.Source.Owner}}
                 Repo: {{$.Source.Repository}}
                 Branch: {{index $.Source.Properties "branch"}}
                 # https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/dynamic-references.html#dynamic-references-secretsmanager
                 # Use the *entire* SecretString with version AWSCURRENT
                 OAuthToken: !Sub
-                    - '{{"{{"}}resolve:secretsmanager:${SecretId}{{"}}"}}'
-                    - { SecretId: !Ref GitHubAccessTokenSecretId }
+                  - '{{"{{"}}resolve:secretsmanager:${SecretId}{{"}}"}}'
+                  - { SecretId: {{$.Source.GitHubPersonalAccessTokenSecretID}} }
```

Comment:
Does this work? it might not matter if we change it to the proposal above to have specific fields :) but I thought this needed to be like this in this case:
```
{{index $.Source.Properties "access_token_secret"}}
```